src/Controls/src/Core/Menu/MenuFlyoutItem.cs

```diff
@@ -1,12 +1,22 @@
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.ComponentModel;
 using System.Windows.Input;
 using Microsoft.Maui.Controls.StyleSheets;
 
 namespace Microsoft.Maui.Controls
 {
-	public partial class MenuFlyoutItem : MenuItem, IMenuFlyoutItem
-	{
-	}
+    public partial class MenuFlyoutItem : MenuItem, IMenuFlyoutItem
+    {
+		public MenuFlyoutItem()
+		{
+			KeyboardAccelerators = new ObservableCollection<KeyboardAccelerator>();
+		}
+
+		public IList<KeyboardAccelerator> KeyboardAccelerators { get; }
+
+        IReadOnlyList<IKeyboardAccelerator>? IMenuFlyoutItem.KeyboardAccelerators => 
+			new List<IKeyboardAccelerator>(KeyboardAccelerators);
```

Comment:
```suggestion
        IReadOnlyList<IKeyboardAccelerator>? IMenuFlyoutItem.KeyboardAccelerators
        {
            if (KeyboardAccelerators?.Count > 0)
                return KeyboardAccelerators.AsReadOnly();
            if (GetAccelerator(this) is IAccelerator acc) // fallback
                return new[] { acc.ToKeyboardAccelerator() };
            return null;
        }
```

Reply:
@PureWeen continuation from above comment

Reply:
AsReadonly seems like a good change.

I wonder if wiring up any aspect of `IAccelerator` is useful. None of that code works in .NET7 so we'd be adding backwards compatibility to something that never really worked in the first place.